tests/test_thread.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+    tests/test_thread.cpp -- call pybind11 bound methods in threads
+
+    Copyright (c) 2017 Laramie Leavitt (Google LLC) <lar@google.com>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#include <chrono>
+#include <thread>
+
+#include <pybind11/cast.h>
```

Comment:
The pybind11 includes must be before the system includes. (I.e. simply move lines 9,10,11 after current line 17.)

Reply:
Done.